articles/android-custom-views-extending-view/index.md

```diff
@@ -0,0 +1,175 @@
+### Introduction
+Android platform has a variety of views. These views tend to meet most applications' UI needs. In some cases, these ready-built views don't conform to the application's requirements. In these cases, developers have to create their own custom views. There are two approaches to creating custom Views. These are:
+
+  1. By extending the androids View class.
+  2. By extending an existing subclass of the View class.
+
+This article will focus on the first approach, i.e., extending the View class. Why extend the View class when you can extend an already built subclass? For instance, in an Android game, most of these applications' views don't resemble any other default view. It shows that the developers most probably built their custom views from scratch. That's what we will go through.
+
+We will create a simple view that is a colored circle with a border then add it to an XML layout.
+
+Let's dive in!
+
+### Prerequisites
+To follow through this tutorial you will need to:
+
+  1. Have [android studio](https://developer.android.com/studio) installed.
+  2. Have a basic knowledge of building android applications.
+  3. Have a basic understanding of Kotlin programming language.
+
+Let's get started!
+
+### Step 1 — Creating An Android Project
```

Comment:
I feel like this step could be summarised since most people who have basic android app development can easily navigate in Android Studio. Try and have it as a paragraph

Reply:
Thanks @LinusMuema.😇 Let me get to it.